src/chttpd/src/chttpd_node.erl

```diff
@@ -46,6 +46,14 @@ handle_node_req(#httpd{method='GET', path_parts=[_, Node, <<"_config">>]}=Req) -
     send_json(Req, 200, {KVs});
 handle_node_req(#httpd{path_parts=[_, _Node, <<"_config">>]}=Req) ->
     send_method_not_allowed(Req, "GET");
+% POST /_node/$node/_config/_reload - Flushes unpersisted config values from RAM
+handle_node_req(#httpd{method='POST', path_parts=[_, Node, <<"_config">>, <<"_reload">>]}=Req) ->
```

Comment:
Do we need to send an explicit `405` for non-POST requests to this endpoint? I tried this:
```
$ curl -u adm:pass -H 'Content-Type: application/json' localhost:15984/_node/_local/_config/_reload -XGET | jq .
{}
```
which doesn't seem right?